content/information-distribution/source.md

```diff
@@ -300,21 +300,26 @@ Matrix](https://matrix.org/docs/projects/try-matrix-now.html#client-sdks).
 
 ```json
 {
-    "algorithm": "ed25519",
-    "sender_key": "IlRMeOPX2e0MurIyfWEucYBRVOEEUMrOHqn/8mLqMjA",
     "signature": "54ab6f6f18d63ef1",
     "signed": {
-        "parent": "@acmenews:example.com",
-        "prev_user": {
-            "user_id": "@acmenewsen:badserver.com",
-            "event_id": "!someEnglishArticle:badserver.com",
-        },
-        "website": "https://www.example.com/en",
-        "description": "This is the English source for ACME News.",
-        "sig_algo": "ed25519",
-        "sig_keys": [
-            "Noh0oot2chahTheixeuviX6seidiweewahK/8mLeMjA"
-        ]
+        "sender": "@acmenews:example.com",
+        "room_id": "!LppXGlMuWgaYNuljUr:example.com",
+        "state_key": "network.informo.subsource",
```

Comment:
That's the event type, not the state key.

Reply:
oh damn...